migrations/20190515180630-updates-is-private.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .addColumn('Updates', 'isPrivate', {
+        type: Sequelize.BOOLEAN,
+        defaultValue: false,
+      })
+      .then(() => {
+        queryInterface.addColumn('UpdateHistories', 'isPrivate', {
```

Comment:
```suggestion
        return queryInterface.addColumn('UpdateHistories', 'isPrivate', {
```

So sequelize can properly await for the migration to be done